pandas/tests/test_format.py

```diff
@@ -3181,6 +3181,16 @@ def test_to_string_float_na_spacing(self):
                     '4       NaN')
         self.assertEqual(result, expected)
 
+    def test_to_string_without_index(self):
+        #GH 11729 Test index=False option
+        s= Series([1, 2, 3, 4])
+        result = s.to_string(index=False)
+        expected = (u(' 1\n') +
+                    ' 2\n' +
```

Comment:
@jorisvandenbossche does this look right? (e.g. I know we add a space, but seems odd with no index....)